Tvl.VisualStudio.InheritanceMargin.CSharp/CSharpInheritanceAnalyzer.cs

```diff
@@ -87,6 +91,9 @@ internal class CSharpInheritanceAnalyzer : BackgroundParser
         private static readonly Lazy<Func<INamedTypeSymbol, Solution, IImmutableSet<Project>, CancellationToken, Task<IEnumerable<INamedTypeSymbol>>>> FindDerivedInterfacesAsync =
             new Lazy<Func<INamedTypeSymbol, Solution, IImmutableSet<Project>, CancellationToken, Task<IEnumerable<INamedTypeSymbol>>>>(() =>
             {
+#if ROSLYN2
+                return (symbol, solution, projects, cancellationToken) => Task.FromResult(Enumerable.Empty<INamedTypeSymbol>());
```

Comment:
@CyrusNajmabadi I currently don't have a Roslyn 2.0 implementation of this method. How would you recommend I implement this  functionality, given that I can no longer easily use the method in `DependentTypeFinder` through reflection (per dotnet/roslyn@a4d3c644cc121300f5b82704ee5f20b5679bb2dc)?

In the current implementation, the behavior of this extension is broken in the following scenario:

```csharp
interface A { }
interface B : A { }
```

The extension *should* show a glyph in the margin where `A` is defined to let the user know that `B` is a derived type. However, `SymbolFinder` appears to only provide methods that work when `B` is a class. :(